Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN docker-php-ext-install mcrypt \
       intl \
       pgsql
 
+RUN pecl install -f memcache
```

Comment:
The `memcache` ext is quite old (deprecated), we should use `memcached` instead I think. Is there any reason to fall back on pecl ? docker-php-ext-install should be able to handle this as well.